VideoApi/VideoApi.Contract/Responses/AddQuickLinkParticipantResponse.cs

```diff
@@ -4,7 +4,7 @@ namespace VideoApi.Contract.Responses
 {
     public class AddQuickLinkParticipantResponse
     {
-        public ParticipantDetailsResponse ParticipantDetails { get; set; }
+        public ParticipantResponse Participant { get; set; }
```

Comment:
just wondering why we need to rename these properties - won't that break the clients?

if you've already updated video web, then no worries

Reply:
I have updated it in the client - changed the name to be more reflective of what the class is, as it doesnt really contain the ParticipantDetails anymore (removed the names, emails, ect..)